test/parallel/test-repl-history-navigation.js

```diff
@@ -554,6 +555,36 @@ const tests = [
     expected: [],
     clean: false
   },
+  {
+    env: { NODE_REPL_HISTORY: defaultHistoryPath },
+    test: ['const util = {}', ENTER,
+           'ut', RIGHT, ENTER],
+    expected: [
+      prompt, ...'const util = {}',
+      'undefined\n',
+      prompt, ...'ut', ' // il', '\n// {}', 'il',
+      '\n// {}',
```

Comment:
```suggestion
      prompt, ...'ut', ...[' // il', '\n// {}',
                           'il', '\n// {}'],
      '{}\n',
```

Reply:
@aduh95 why is 
```
...[' // il', '\n// {}',
                           'il', '\n// {}'],
```
needs to be in an array and not laid out directly?

Reply:
It doesn't need to be, I just prefer the alignement; feel free to remove the array if you prefer without.